docker-compose.yml

```diff
@@ -62,6 +62,38 @@ services:
     environment:
       VNC_NO_PASSWORD: "true"
 
+  middleware-oc10:
+    image: dpakach/owncloud-test-middleware:latest
+    hostname: middleware
+    extra_hosts:
+      - host.docker.internal:${DOCKER_HOST:-host-gateway}
+    ports:
+      - "3000:3000"
+    volumes:
+      - /var/tmp/ocis:/var/tmp/ocis
+      - ./tests/acceptance/filesForUpload:/usr/src/app/filesForUpload
+    environment:
+      MIDDLEWARE_PORT: 3000
```

Comment:
Can we use different ports here so we can start both the ocis-middleware and the oc10-middleware at the same time? :)